instrumentation-genai/opentelemetry-instrumentation-openai-v2/README.rst

```diff
@@ -10,6 +10,17 @@ This library allows tracing LLM requests and logging of messages made by the
 `OpenAI Python API library <https://pypi.org/project/openai/>`_. It also captures
 the duration of the operations and the number of tokens used as metrics.
 
+Many LLM platforms support the OpenAI SDK. This means systems such as the following are observable with this instrumentation when accessed using it:
+
+| Name  | gen_ai.system |
```

Comment:
Is this a markdown table? This is a rst document

Reply:
good catch, thanks @xrmx

Reply:
I keep forgetting.. my fault 😊